### DISCUSSION TITLE
Support templated headings for daily note calendars

### DISCUSSION THREAD

**Discussion by wc8** [davish/obsidian-full-calendar#417] [Feature Requests]
Is there a way to write the heading for a Daily Notes calendar to parse  
`# [[<% tp.date.now("YYYY-MM-DD dddd", -1) %>|⇚]] **<% tp.date.now("YYYY MMM DD dddd", 0) %>** [[<% tp.date.now("YYYY-MM-DD dddd", 1) %>|⇛]] ` 
into the correct heading?

So, for example, today, new events/tasks would land under:
 `# [[2023-02-28 Tuesday|⇚]] **[[2023 Mar 01 Wednesday]]** [[2023-03-02 Thursday|⇛]] `
rather than under a heading with the templater code?

**Comment by davish** [comment]
This is not possible right now. I'll tag this as an enhancement.
